pydantic/small_molecule.json

```diff
@@ -0,0 +1,135 @@
+{
+  "$defs": {
+    "SmallMoleculeBatch": {
+      "properties": {
+        "smallMoleculeLabBatchLabel": {
+          "description": "Lab-specific ID for the batch of small molecule used in the experiment.",
+          "title": "Lab Batch Label",
+          "type": "string"
+        },
+        "smallMoleculeSmilesBatch": {
+          "description": "SMILES representation only as provided from the vendor; full structure with all information, i.e. isomeric SMILES",
+          "title": "Small Molecule Vendor-provided SMILES",
+          "type": "string"
+        },
+        "smallMoleculeProviderName": {
+          "description": "Vendor or lab that supplied the small molecule",
+          "title": "Small Molecule Provider Name",
+          "type": "string"
+        },
+        "smallMoleculeProviderCatalogId": {
+          "description": "ID or catalogue number assigned by the vendor or provider to the small molecule",
+          "title": "Small Molecule Provider Catalog ID",
+          "type": "string"
+        },
+        "smallMoleculeProviderBatchId": {
+          "description": "SMILES representation only as provided from the vendor; full structure with all information, i.e. isomeric SMILES",
+          "title": "Small Molecule Vendor-provided SMILES",
+          "type": "string"
+        }
+      },
+      "title": "SmallMoleculeBatch",
+      "type": "object"
+    },
+    "SmallMoleculeCanonical": {
+      "properties": {
+        "smallMoleculeName": {
+          "description": "The common, primary, recognizable name for the small molecule being used.",
+          "title": "Small Molecule Name",
+          "type": "string"
+        },
+        "smallMoleculeAlternativeNames": {
+          "description": "List of synonymous compound names, drug name (if applicable), and other alternative names",
+          "items": {
+            "type": "string"
+          },
+          "title": "Small Molecule Alternative Names",
+          "type": "array"
+        },
+        "smallMoleculeLabName": {
```

Comment:
why there is Lab Name on Canonical Metadata? And even for batches, for me those should be managed by application ownership and permission system, why I would need to input this 150 times, if I am always managing datasets from the same lab

Reply:
I'll change the name to small_molecule_batch.json

I agree about Lab Name, I think we can remove the Lab Name field entirely if it can be retrieved from the foreign key for the owner.